### ISSUE TITLE
bad argument #2 to 'format' (no value)

### ISSUE THREAD

**Issue by nagels** [esx-framework/es_extended#1170]
## Bug Report

> Please use the template below and replace all example lines with your issue!

## Issue Checklist

> Please put `x` inside of the box that matches your issue.

- [x ] I'm using the latest version (develop branch).
- [ x] The issue has occured more than once.
- [ x] I have followed the readme and my setup is correct.
- [ x] I have searched the ESX forum and Github issues for solution before submitting my issue.
- [ x] I'm certain this is an issue with es_extended and not my personal module / script.

## Expected Result

> In a few short sentences please tell us what the expected result / correct behaviour should occur.

The player should be revived.

## Current Result

> In a few short sentences please tell us what is currently happening and isn't "normal" (Images are strongly recommended).

Using the ESX_Ambulancejob, when reviving a player with a medikit an error in console pops up everytime.
The animation and notification from the ambulance jobs pops up telling me that everything should be fine.

## How to reproduce the problem

> What steps did you take for the issue to occur? What would we have to do for the same result to happen?

1. Shoot a player
2. Get an EMS to use the revive action from the menu
3. look for the error in console
## Errors

> Screenshot or direct copy and paste of the error from the console window.

`[script:esx_ambulance] SCRIPT ERROR: @es_extended/locale.lua:8: bad argument #2 to 'format' (no value)`
https://gyazo.com/5ac856ac71dd531d18f5ba0cf9720deb

### Code Block referenced by the error

> If the code references a specific line of code within a file, what line is it?
Line 8
`Locales = {}

function _(str, ...)  -- Translate string

	if Locales[Config.Locale] ~= nil then

		if Locales[Config.Locale][str] ~= nil then
			return string.format(Locales[Config.Locale][str], ...)
		else
			return 'Translation [' .. Config.Locale .. '][' .. str .. '] does not exist'
		end

	else
		return 'Locale [' .. Config.Locale .. '] does not exist'
	end

end

function _U(str, ...) -- Translate string first char uppercase
	return tostring(_(str, ...):gsub("^%l", string.upper))
end

function LoadLocale(ns, lang, data)

  Locales[lang] = Locales[lang] or {}

  for k,v in pairs(data) do
    Locales[lang][ns .. ':' .. k] =v
  end

end`

### Possible Solution

> If you have an idea of a potential solution or have a suggestion of how to fix the issue please write it below.

I know this is not the place but.
I've already tried replacing both locale files from each script without any luck.
Server has been restarted and cache cleared.

**Comment by thelindat** [comment]
Check which locale is being triggered and the format; then check what is being sent to it. This is not an ESX specific issue.

**Comment by nagels** [comment]
It's getting triggered from esx_ambulance would be my guess.
It comes when doing the revive through the EMS job.

**Comment by thelindat** [comment]
> 
> 
> It's getting triggered from esx_ambulance would be my guess.
> It comes when doing the revive through the EMS job.

Yes, so look which locale is triggered and confirm what is being sent.
If the translation contains `%s` twice then it needs two vars to be passed to it
